### PR TITLE
Add weight query

### PR DESCRIPTION
**Pull request by alemf**

### CODE REVIEW

**Comment by arnauddrain** [comment]
Hello @alemf , thank you so much for your contribution! 🙏 
In order to ease the review, I added prettier to the project and standardize all of the line endings, could you merge the modifications in your branch? (feel free to add me as a contributor if you don't want to handle the merge conflict)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+### Local Setup
+
+1. Fork and clone the repo.
+1. Install the dependencies.
+
+    ```shell
+    npm install
+    ```
+
+1. Install SwiftLint if you're on macOS.
```
Comment (arnauddrain) [inline CONTRIBUTING.md]: Is there a point to add references to Swift in a plugin that does only work on Android?

File: CapacitorGoogleFit.podspec [hunk 2]
```diff
@@ -0,0 +1,17 @@
+require 'json'
```
Comment (arnauddrain) [inline CapacitorGoogleFit.podspec]: Is there a point to add a podspec file in a plugin that does only work on Android?

File: android/src/main/java/com/adscientiam/capacitor/googlefit/GoogleFitPlugin.java [hunk 3]
```diff
@@ -0,0 +1,292 @@
+package com.adscientiam.capacitor.googlefit;
```
Comment (arnauddrain) [inline android/src/main/java/com/adscientiam/capacitor/googlefit/GoogleFitPlugin.java]: This file is duplicated, could you merge it with the other one?

**Review by arnauddrain — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (alemf) [reply CONTRIBUTING.md]: Hi this file is autogenerated I removed it.

File: CapacitorGoogleFit.podspec (hunk 2 above)
Comment (alemf) [reply CapacitorGoogleFit.podspec]: This file was also auto-generated i removed it.

File: android/src/main/java/com/adscientiam/capacitor/googlefit/GoogleFitPlugin.java (hunk 3 above)
Comment (alemf) [reply android/src/main/java/com/adscientiam/capacitor/googlefit/GoogleFitPlugin.java]: I removed one of these files

**Comment by alemf** [comment]
> Hello @alemf , thank you so much for your contribution! 🙏 In order to ease the review, I added prettier to the project and standardize all of the line endings, could you merge the modifications in your branch? (feel free to add me as a contributor if you don't want to handle the merge conflict)

Hi,
I merged your changes into my branch, thank you for the fast reply. I didn't have the time to work on this earlier.

**Comment by alemf** [comment]
Hi @arnauddrain,
Same question here, do you think you're going to have time to review this pull request?

Greetings :)

**Comment by arnauddrain** [comment]
> Hi @arnauddrain, Same question here, do you think you're going to have time to review this pull request?
> 
> Greetings :)

Hello @alemf ,  I don't think I will have the time to maintain properly this extension, as I am not working on this subject anymore.
I will be glad to give you access to the npm package so you can keep the name. And mark this repo as deprecated with a link to your fork.
When you're ready, can you give me your npm username and a link to your fork?

**Pull request closed without merging**